### PR TITLE
channel name bifurcation

### PR DESCRIPTION
**Pull request by deekshithepns**

### CODE REVIEW

**Review by corlard3y — DISMISSED** [review]

File: src/components/ChannelInfo.tsx [hunk 1]
```diff
@@ -76,52 +84,52 @@ const ChannelInfo = ({
...
-                value={networkName[chainDetails]}
-              />
-            </DropdownStyledParent>
+          {/* dropdown */}
+          <DropdownStyledParent>
+            <DropdownStyled
+              options={ALIAS_CHAINS}
+              onChange={(option) => {
+                setChainDetails(option.value);
+                console.log(option);
```
Comment (mishramonalisha76) [inline src/components/ChannelInfo.tsx:128]: remove console log

File: src/components/ChannelInfo.tsx [hunk 2]
```diff
@@ -76,52 +84,52 @@ const ChannelInfo = ({
...
-              <DropdownStyled
-                options={ALIAS_CHAINS}
-                onChange={(option) => {
-                    setChainDetails(option.value);
-                    console.log(option);
-                }}
-                value={networkName[chainDetails]}
-              />
-            </DropdownStyledParent>
+          {/* dropdown */}
```
Comment (mishramonalisha76) [inline src/components/ChannelInfo.tsx:122]: commented code can be removed

File: src/components/ChannelInfo.tsx [hunk 3]
```diff
@@ -76,52 +84,52 @@ const ChannelInfo = ({
...
+          margin="7px 0px 0px 0px"
+          flex="1"
+          self="stretch"
+          align="stretch"
+          border="1px solid #BAC4D6"
+          radius="12px"
+        >
+          <Input
+            required
+            // placeholder="Your Channel Name"
```
Comment (mishramonalisha76) [inline src/components/ChannelInfo.tsx:99]: we can remove this line

File: src/components/ChannelInfo.tsx [hunk 4]
```diff
@@ -139,172 +147,185 @@ const ChannelInfo = ({
...
-              <Span color={theme.default.secondaryColor} size="13px" margin="0px 10px 0px 0px" weight='700'>{250 - channelInfo.length}</Span>
-            </Item>
-            <TextField
+            <Label style={{ color: theme.color }}>Channel Alias address</Label>
+            <Input
               required
-              // placeholder="Your Channel's Short Description (250 Characters)"
-              rows="4"
-              maxlength="250"
+              // placeholder="Your Channel Address"
```
Comment (mishramonalisha76) [inline src/components/ChannelInfo.tsx:162]: we can remove this line

File: src/components/ChannelInfo.tsx [hunk 5]
```diff
@@ -139,172 +147,185 @@ const ChannelInfo = ({
...
+              size="13px"
+              margin="0px 10px 0px 0px"
+              weight="700"
+            >
+              {250 - channelInfo.length}
+            </Span>
           </Item>
+          <TextField
+            required
+            // placeholder="Your Channel's Short Description (250 Characters)"
```
Comment (mishramonalisha76) [inline src/components/ChannelInfo.tsx:218]: remove this line

**Review by mishramonalisha76 — CHANGES_REQUESTED** [review]

**Review by mishramonalisha76 — APPROVED** [review]

**Pull request merged**